### PR TITLE
Updated README file for 2.0 Examples

### PR DESCRIPTION
**Pull request by ATweedMITRE**
--Completely redid the README file to highlight comments on the examples
--Removed unnecessary examples that are being changed for 2.0
--Added using-marking-definitions.json example

### CODE REVIEW

File: examples/using-marking-definitions.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "type": "identity",
+      "id": "identity--611d9d41-dba5-4e13-9b29-e22488058ffc",
+      "created": "2017-04-14T13:07:49.812Z",
+      "modified": "2017-04-14T13:07:49.812Z",
+      "name": "Stark Industries",
+      "identity_class": "organization",
+      "contact_information": "info@stark.com",
+      "sectors": [
+        "defense"
+      ],
```
Comment (clenk) [inline examples/using-marking-definitions.json]: Delete this comma to make the JSON well-formed. I think this is what caused the Travis error.

Comment (gtback) [inline examples/using-marking-definitions.json]: Thanks for figuring this out, @clenk ! I agree that that seems to be the cause.

**Comment by ATweedMITRE** [comment]
Apologies on the typo. Should be good to go now. Also, I added quotes around SHA-256 where need be in the examples based on the changes in WD02.

**Comment by gtback** [comment]
Looks like I need to update the validator to use the latest version of the pattern grammar.

**Comment by gtback** [comment]
I released new versions of the pattern-validator and stix-validator, which take care of most of the issues in here. @ATweedMITRE, can you merge in the master branch and fix the one remaining thing (the `phishing` label is the granular-marking example is invalid).

**Pull request merged**